cni-plugin/integration/testutil/test_util.go

```diff
@@ -39,6 +40,70 @@ func (r *TestRunner) walkConfDir() (map[string]struct{}, error) {
 	return fileNames, nil
 }
 
+// Checks that the embedded linkerd config is of the expected form
+// and contains the right values:
+//
+//	"linkerd": {
+//	   "incoming-proxy-port": 4143,
+//	   "outgoing-proxy-port": 4140,
+//	   "proxy-uid": 2102,
+//	   "ports-to-redirect": [],
+//	   "inbound-ports-to-ignore": ["4191","4190"],
+//	   "simulate": false,
+//	   "use-wait-flag": false
+//	 }
+func checkLinkerdCniConf(wrapperConf map[string]any) error {
+	var conf = wrapperConf["linkerd"].(map[string]any)
+
+	var incomingProxyPort = conf["incoming-proxy-port"].(float64)
+	if incomingProxyPort != 4143 {
+		return fmt.Errorf("incoming-proxy-port has wrong value: expected: %v, found: %v",
```

Comment:
```suggestion
		return fmt.Errorf("incoming-proxy-port has wrong value, expected: %v, found: %v",
```

Just a tiny nitpick, noticed the rest use a comma after.